Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:36 AS builder
+FROM registry.fedoraproject.org/fedora:37 AS builder
```

Comment:
This needs to update the second `FROM` below too.

Reply:
Thank you for catching that, I am making a pr now.